src/js/components/FormField/__tests__/FormField-test.js

```diff
@@ -362,4 +362,45 @@ describe('FormField', () => {
     const tree = component.toJSON();
     expect(tree).toMatchSnapshot();
   });
+
+  test('should render asterisk when requiredIndicator === true', () => {
```

Comment:
We are trying to stay away from using `renderer`, please take a look at the test pattern of 
`const { container } = render(...)` instead, and see if it can work as well. 
It will keep us more up to date with using `@testing-library/react`.

Reply:
Yes, I'll update the tests accordingly.

Reply:
Done.